mods/beds/functions.lua

```diff
@@ -97,13 +99,13 @@ local function update_formspecs(finished)
 
 	if finished then
 		form_n = beds.formspec ..
-			"label[2.7,11; Good morning.]"
+			S("label[2.7,11; Good morning.]")
 	else
 		form_n = beds.formspec ..
-			"label[2.2,11;"..tostring(player_in_bed).." of "..tostring(ges).." players are in bed]"	
+			"label[2.2,11;"..tostring(player_in_bed)..S(" of ")..tostring(ges)..S(" players are in bed]")	
```

Comment:
This should be better implemented via passing multiple parameters to S, like `S("@1 of @2 players are in bed", tostring(player_in_bed), tostring(ges))`. There is also a way where there is no `tostring`, but I can't remember how, and intllib is badly documented in this regard. I think @ShadowNinja knows how.